Connector/Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM python:3.7
 
 WORKDIR /usr/src/app
+COPY ./Connector ./Connector
+COPY .availableCurrencies.json ./.availableCurrencies.json
```

Comment:
If you put COPY instructions below RUN instructions you would avoid reinstalling all python dependencies for each code change in Connector files. I know you need file requirements.txt for python install but I think could be better: 

1. COPY only requirements file
2. Install dependencies
3. COPY rest of Connector folder files

This approach is used in Dockers with NodeJS for caching node_modules instead of reinstalling it, I think it could be a good performance improvement here too, but I'm not sure if this task is part of the scope of your PR so it's only a suggestion

Reply:
Yep this would be an improvement for this dockerfile. We aim to improve the performance of every dockerfile and docker-compose in the 4rd month @hippocampusSwapper